system/CLI/BaseCommand.php

```diff
@@ -212,7 +216,7 @@ public function getPad(array $array, int $pad): int
     /**
      * Makes it simple to access our protected properties.
      *
-     * @return array|Commands|LoggerInterface|string|null
+     * @return Commands|list<mixed>|LoggerInterface|string|null
```

Comment:
Why `list<mixed>`? Isn't it `array<string, string>`?

Reply:
When I ran the `composer cs-fix` command, I encountered an error. Following the information at https://phpstan.org/writing-php-code/phpdoc-types, (`mixed` type can be used if we don’t want to define a more specific type.  PHPStan doesn’t check anything on the mixed type - any property or method can be called on it and it can be passed to any type in a function/method call.) I updated it to `list<mixed>`. However, after the warning, I reviewed it again and realized that the error actually stemmed from the doc type ordering in /CodeIgniter4/system/CLI/BaseCommand.php (phpdoc_types_order). Change to `array<string, string>`. Thanks.

Reply:
We should define specific types if we can.